tests/Collection/CollectionFunctionalTest.php

```diff
@@ -381,7 +381,9 @@ public function testWithOptionsPassesOptions()
 
     /**
      * @group matrix-testing-server-4.4-driver-4.0
+     * @group matrix-testing-server-4.4-driver-4.2
      * @group matrix-testing-server-5.0-driver-4.0
+     * @group matrix-testing-server-5.0-driver-4.2
```

Comment:
I realize this format dates back to https://github.com/mongodb/mongo-php-library/pull/810/files#r579294457 (already merged in v1.4), but seeing this again I wonder if it makes sense to add something to this name that communicates the tests are actually skipped for this server/driver combination.

My first impression reading this after not thinking about it for some time is that these are tests _for_ this particular version, not incompatible tests intended to be skipped.

Maybe just prefix "exclude-" to these?

Reply:
Adding a prefix sounds good. I'll make the change in 1.4 and merge up, fixing things along the way.